regions/io/crtf/test/data/CRTFgeneral.reg

```diff
@@ -0,0 +1,20 @@
+#CRTF0
+global coord=B1950_VLA, frame=BARY, corr=[I, Q], color=blue
+
+# A simple circle region:
+circle[[18h12m24s, -23d11m00s], 2.3arcsec]
+
+# A box region, this one only for annotation:
+ann box[[140.0342deg, -12.34243deg], [140.0360deg, -12.34320deg]]
+
+# A rotated box region, for a particular range of velocities:
+rotbox[[12h01m34.1s, 12d23m33s], [3arcmin, 1arcmin], 12deg], range=[-1240km/s, 1240km/s]
+
+# An annular region, overriding some of the global defaults:
+annulus[[17h51m03.2s, -45d17m50s], [0.10deg, 4.12deg]], corr=[I,Q,U,V], color=red, label='My label here'
+
+# Cuts an ellipse out of the previous regions, but only for Q and a particular frequency range:
+-ellipse[[12deg, +45deg], [0.25deg, 1.34deg], 45rad], range=[1.420GHz, 1.421GHz], corr=[Q], color=green, label='Removed this'
+
+# A diamond marker, in J2000 coordinates:
+symbol[[32.1423deg, 12.1412deg], D], linewidth=2, coord=J2000, symsize=2
```

Comment:
@keflavich This is the only official sample I am having now. 
Though I can come up with basic test cases of my own.
Like I cannot be sure about the reliability of the Parser until we can have some practical CRTF files. (Like the dec representation as `-32.6.7.8` as we discussed earlier.)

Reply:
I'm starting to search out more examples.  But one straightforward and important thing to do is create round-trip tests to and from ds9 regions.  So, for example, take a ds9 region file, convert each region to its CRTF equivalent, then read it back and convert to ds9 and make sure it's the same.

Reply:
I still haven't made methods for conversion between CRTF and DS9 . I will make sure I complete that and perform round tests.